libcodechecker/session_manager.py

```diff
@@ -57,18 +57,22 @@ def calc_persistency_hash(auth_string):
         return hashlib.sha256(auth_string + "@" +
                               _Session.__initial_salt).hexdigest()
 
-    def __init__(self, token, phash):
+    def __init__(self, token, phash, user, authenticated):
+        self.authenticated = authenticated
+        self.user = user
         self.token = token
         self.persistent_hash = phash
         self.last_access = datetime.now()
+        self.destroyed = False
```

Comment:
Why do we have both a list of valid sessions and an instance variable if a session is destroyed?